crowbar_framework/config/locales/crowbar/en.yml

```diff
@@ -797,8 +797,8 @@ en:
           help:
             default: 'Make sure Ceph is healthy'
         clusters_health:
-          crm_failures: '"crm status" failed at nodes %{nodes}.'
-          failed_actions: '"crm status" is showing some failed actions at nodes %{nodes}.'
+          crm_failures: '"crm status" failed. Please check that all cluster resources are online or refer to the SUSE High Availability documentation for possible troubleshooting.'
+          failed_actions: '"crm status" is showing some failed actions. Please recheck the cluster health with "crm_status -C" or refer to the SUSE High Availability documentation for possible troubleshooting.'
```

Comment:
yeah, need to recheck this as I didnt have the env available when creating the pr, I did eyeball it, so it may be `crm status -D`